### PR TITLE
Add issue and PR templates and CODEOWNERS file

### PR DESCRIPTION
**Pull request by Pietfried**

### CODE REVIEW

**Comment by Pietfried** [comment]
Screenshot from 2024-02-02 17-47-01
Screenshot from 2024-02-02 17-46-46

**Comment by lategoodbye** [comment]
Hi,
just two possible ideas for the bug report form:
- as it's possible to compile libocpp with different compile options this should be mention under "To Reproduce"
- i think this currently not a problem but soon it will become one - Disclaimer note that the reporter already searched for the issue before creating a new one

File: README.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ Libocpp can be used for the communication of one charging station and multiple E
 
 Libocpp provides a complete implementation of OCPP 1.6. The implementation of OCPP 2.0.1 is currently under development.
 
+## Get Involved
+
+See the [COMMUNITY.md](COMMUNITY.md) and [COMMUNITY.md](COMMUNITY.md) of the EVerest project to get involved.
```
Comment (barsnick) [inline README.md:12]: These are two links to the same file.

The file is located in a different repo, link needs to be adapted: https://github.com/EVerest/EVerest/blob/main/COMMUNITY.md

**Comment by marcemmers** [comment]
This is going to be really nice! 

Can it automatically set tags as well? Based on the issue type and maybe also the dropdown selection?

I think the feature request can use some tweaking. It seems to be to focus mainly on the what and also on the how. I would suggest making required a "what is the problem/need" and "why is it a problem/need" since that really helps you to know what is requested. Those two could be in a single field as well. 

Then an optional field that leaves room for a suggestion on the how. That leaves room for the implementer to also look at the problem and find a good solution, potentially with the suggestion.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## Describe your changes
+
+## Issue ticket number and link
+
+## Checklist before requesting a review
+- [ ] I have performed a self-review of my code
+- [ ] My code follows the style guidelines of this project
+- [ ] I have made corresponding changes to the documentation
```
Comment (DerMolly) [inline .github/pull_request_template.md:8]: We could add a `CONTRIBUTING.md` explaining things like DCO and let the PR creator click a checkbox here that they have a read that:

```suggestion
- [ ] I have made corresponding changes to the documentation
- [ ] I read the [contribution documentation](https://github.com/EVerest/libocpp/blob/main/CONTRIBUTING.md) and
  made sure that:
  - My commits are signed-off to accept the DCO.
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+- [ ] I read the [contribution documentation](https://github.com/EVerest/libocpp/blob/main/CONTRIBUTING.md) and made sure that my changes meet its requirements
```
Comment (Pietfried) [inline .github/pull_request_template.md:9]:
```suggestion
- [ ] I read the [contribution documentation](https://github.com/EVerest/EVerest/blob/main/CONTRIBUTING.md) and made sure that my changes meet its requirements
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,2 @@
+* @pietfried @hikinggrass @marcemmers @maaikez
+/.github/ @pietfried
```
Comment (Pietfried) [inline .github/CODEOWNERS:2]:
```suggestion
/.github/ @pietfried @hikinggrass
```

**Review by hikinggrass — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    attributes:
+      label: Describe the problem
+      description: What problem is your feature request targeting and why is it a problem? Please describe.
+      placeholder: |
+        A clear and concise description of what the problem is.
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
```
Comment (Pietfried) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:28]:
```suggestion
    id: solution
```

**Pull request merged**